salt/states/docker_container.py

```diff
@@ -2096,21 +2095,20 @@ def _get_nets():
     return _format_comments(ret, comments)
 
 
-def run(name,
-        image=None,
-        onlyif=None,
-        unless=None,
-        creates=None,
-        bg=False,
-        failhard=True,
-        replace=False,
-        force=False,
-        skip_translate=None,
-        ignore_collisions=False,
-        validate_ip_addrs=True,
-        client_timeout=salt.utils.docker.CLIENT_TIMEOUT,
-        **kwargs):
-    '''
+def run(
+    name,
+    image=None,
+    bg=False,
+    failhard=True,
+    replace=False,
+    force=False,
+    skip_translate=None,
+    ignore_collisions=False,
+    validate_ip_addrs=True,
+    client_timeout=salt.utils.dockermod.CLIENT_TIMEOUT,
```

Comment:
Since we haven't backported https://github.com/saltstack/salt/pull/56857
then, this should read `salt.utils.docker.CLIENT_TIMEOUT`

Currently, it is causing `AttributeError: 'module' object has no attribute 'dockermod'`
in the CI's.

Reply:
oh, yes. Thanks for catching it. Will take care of it.

Reply:
PRs for `3000` and `3000.3` created to fix this regression: https://github.com/openSUSE/salt/pull/421 https://github.com/openSUSE/salt/pull/422